_posts/2016-10-04-omero-5.2.6.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Release of OMERO 5.2.6
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.2.6
+---
+
```

Comment:
The example at https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files
has no leading whitespace line.